src/gui/accelerators.c

```diff
@@ -617,26 +614,23 @@ void dt_accel_widget_toast(GtkWidget *widget)
         break;
     }
 
-    if (show)
-    {
-      if(w->label[0] != '\0')
-      { // label is not empty
-        if(w->module && w->module->multi_name[0] != '\0')
-          dt_toast_log(_("%s %s / %s: %s"), w->module->name(), w->module->multi_name, w->label, text);
-        else if(w->module && !strstr(w->module->name(), w->label))
-          dt_toast_log(_("%s / %s: %s"), w->module->name(), w->label, text);
-        else
-          dt_toast_log(_("%s: %s"), w->label, text);
-      }
+    if(w->label[0] != '\0')
+    { // label is not empty
+      if(w->module && w->module->multi_name[0] != '\0')
+        dt_toast_log(_("%s %s / %s: %s"), w->module->name(), w->module->multi_name, w->label, text);
+      else if(w->module && !strstr(w->module->name(), w->label))
+        dt_toast_log(_("%s / %s: %s"), w->module->name(), w->label, text);
       else
-      { //label is empty
-        if(w->module && w->module->multi_name[0] != '\0')
-          dt_toast_log(_("%s %s / %s"), w->module->name(), w->module->multi_name, text);
-        else if(w->module)
-          dt_toast_log(_("%s / %s"), w->module->name(), text);
-        else
-          dt_toast_log(_("%s"), text);
-      }
+        dt_toast_log(_("%s: %s"), w->label, text);
+    }
+    else
+    { //label is empty
+      if(w->module && w->module->multi_name[0] != '\0')
+        dt_toast_log(_("%s %s / %s"), w->module->name(), w->module->multi_name, text);
+      else if(w->module)
+        dt_toast_log(_("%s / %s"), w->module->name(), text);
+      else
+        dt_toast_log(_("%s"), text);
```

Comment:
we don't need the `_(...)` here as there is nothing to translate Above it is different as there is the `/` or `:` that could be changed or spaced differently.